include/boost/geometry/views/reversible_view.hpp

```diff
@@ -53,19 +90,14 @@ struct reversible_view {};
 template <typename Range>
 struct reversible_view<Range, iterate_forward>
 {
-    typedef identity_view<Range> type;
+    using type = identity_view<Range>;
 };
 
 
 template <typename Range>
 struct reversible_view<Range, iterate_reverse>
```

Comment:
Ah, it's still here.

Reply:
Yes, these metafunctions are here only for backward compatibility.